acceptance/Rakefile

```diff
@@ -26,10 +30,16 @@ module HarnessOptions
     :repo_proxy => true,
     :add_el_extras => true,
     :preserve_hosts => 'onfail',
-    :forge_host => 'forge-aio01-petest.puppetlabs.com',
-    :'master-start-curl-retries' => 30,
+    :'master-start-curl-retries' => 30
+  }
+
+  site_specific_info = {
+    :epel_url   => 'http://osmirror.delivery.puppetlabs.net/fedora-epel',
+    :forge_host => 'forge-aio01-petest.puppetlabs.com'
```

Comment:
The forge host is controlled by the env var `BEAKER_FORGE_HOST` with the above as a default. 

Looking at it, I realize it's that way because there's no valid value for it outside of our network.

Since the forge host is needed for site specific tests and there's a Beaker env var already, on second thought I don't actually think I should be changing around the forge_host configuration.